cfme/configure/red_hat_updates.py

```diff
@@ -90,10 +90,11 @@ def make_update_button(text):
 }
 
 
-appliances_table = lambda: version.pick({
-    version.LOWEST: CheckboxTable("//div[@id='form_div']/table[@class='style3']"),
-    '5.4': CheckboxTable("//div[@id='form_div']/table")
-})
+def appliances_table():
```

Comment:
im starting to see more and more of those detached version picks, i do wonder if a abstraction would help